mintpy/__main__.py

```diff
@@ -58,452 +58,452 @@ def _autocomplete(parser):
 # in alphabetical order
 # A-G
 def get_add_parser(subparsers=None):
-    from . import add
+    from .cli import add
```

Comment:
Any reason we're preferring relative imports over absolute? Generally the python community/best practices prefer absolute as it makes the package namespace a lot clearer, but does require a little more work when refactoring (though any good IDE will handle all that work for you).

Reply:
no, just my habit.
I will change all imports to absolute ...

Reply:
Done!